optaplanner-examples/src/main/resources/org/optaplanner/examples/machinereassignment/solver/machineReassignmentSolverConfig.xml

```diff
@@ -9,6 +9,10 @@
   <scoreDirectorFactory>
     <incrementalScoreCalculatorClass>org.optaplanner.examples.machinereassignment.solver.score.MachineReassignmentIncrementalScoreCalculator</incrementalScoreCalculatorClass>
     <!--<scoreDrl>org/optaplanner/examples/machinereassignment/solver/machineReassignmentScoreRules.drl</scoreDrl>-->
+    <!-- TODO incrementalScoreCalculator and scoreDRL give different scores, so assertionScoreDirectorFactory fails -->
```

Comment:
This issue was already there, I just exposed it with a TODO for now. It's not in scope of fixing that for the RHDM-273 jira.